conans/client/rest/conan_requester.py

```diff
@@ -7,6 +7,7 @@
 
 import requests
 from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.ssl_ import create_urllib3_context
```

Comment:
A safer/easier approach would be to move this import inside `init_poolmanager` and handle the `ImportError` too.

Reply:
I am concerned about one thing. If we provide a fallback, that quietly omit using the ssl store because the requests version that is installed in the system is imcompatible with this feature, then suddenly we will have different behavior (and a difficult one to debug) in different machines, just because something in the environment is different.

I don't know which would be the best approach if forcing a requests>2.16.2, can we even do that without breaking other users? Should at least a warning be displayed? I know this is a bit frustrating for such a small change, but it can have an impact on a large number of users if we don't carefully consider different angles.

Reply:
I think that a warning or an info message could be enough ("using/not using system trust stores").

There are already a lot of things from the environment that will lead to different behavior: current `requests` range (2.8.1 - 3.0.0) and `urllib3` range, env variables like `REQUESTS_CA_BUNDLE` or `CURL_CA_BUNDLE`,... and I think that this specific failure is not the harder one to debug, the error message should contain something about "SSL: CERTIFICATE_VERIFY_FAILED" although I agree that sometimes it's not easy to fix.